ssl4rs/data/metadata/disa.py

```diff
@@ -168,7 +168,7 @@
     "field_geoms",  # since both polygon counts and point counts can vary across locations
     "image_order_ids",  # list of strings/identifiers
     "image_metadata",  # list of dicts with lots of more-or-less useful stuff in them
-    "image_data",  # array whose first dimension varies (it's the number of matched orders)
+    #"image_data",  # array whose first dimension varies (it's the number of matched orders)
```

Comment:
not sure why this is getting commented here? (likely breaks code elsewhere, this needs to be changed)